bindings/python/pymongoarrow/schema.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, schema):
     def __iter__(self):
         yield from self.typemap
 
+    def __repr__(self):
+        return str(self.typemap)
```

Comment:
Let's follow the python docs recommendation for repr:
> Return a string containing a printable representation of an object. For many types, this function makes an attempt to return a string that would yield an object with the same value when passed to [eval()](https://docs.python.org/3/library/functions.html#eval); otherwise, the representation is a string enclosed in angle brackets that contains the name of the type of the object together with additional information often including the name and address of the object.

Reply:
See pymongo's codebase for some examples, like: https://github.com/mongodb/mongo-python-driver/blob/b1939e147073ad601666820a1b94af45949a8a27/pymongo/server.py#L292-L293

Reply:
Done. In one of my new commits.